tests/OTAnalytics/application/analysis/test_traffic_counting.py

```diff
@@ -670,6 +674,121 @@ def create_counting_test_cases(self) -> list[tuple]:
             ),
         ]
 
+    def build_max_duration_test_cases(
+        self,
+    ) -> list[tuple[list[FlowCandidate], FlowCandidate]]:
+        return [
+            self.__create_same_duration_candidates(),
+            self.__create_same_duration_candidates(),
```

Comment:
These are the same candidates. Did you actually want to use `__create_different_durration_candidates` instead?